exchanges/bybit/bybit_wrapper.go

```diff
@@ -433,7 +423,7 @@ func (by *Bybit) UpdateTickers(ctx context.Context, assetType asset.Item) error
 		}
 
 		for x := range ticks.List {
-			pair, err := avail.DeriveFrom(ticks.List[x].Symbol)
+			pair, enabled, err := by.MatchSymbolCheckEnabled(ticks.List[x].Symbol, assetType, false)
 			if err != nil {
```

Comment:
Its hidden by a lot of lame stuff (mock tests + `UpdateTradablePairs` in `TestMain`), but this also has the same potential issue